### DISCUSSION TITLE
ant.design官网比github还难打开

### DISCUSSION THREAD

**Discussion by yyyuping** [ant-design/ant-design#36391] [General]
在不科学上网的情况下，ant.design官网比github还难打开。能优化下吗，想看个设计文档也太艰难了

**Comment by afc163** [comment]
<img width="435" alt="图片" src="https://user-images.githubusercontent.com/507615/177501295-d45aa103-6fe3-45ec-99f6-f834a2d3e7ee.png">

**Reply by yyyuping** [reply]
已解决，感谢回复:-)
